File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int i = s.length() - 1;
        int count = 0;

        while (i >= 0 && isspace(s[i])) {
            i -= 1;
        }

        while (i >= 0 && !isspace(s[i])) {
            count++;
            i -= 1;
        }

        return count;
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        count = 0
        if not s:
            return 0
        for i in range(len(s) - 1, -1, -1):
            if s[i] == " ":
                continue
            if s[i] != " ":
                count += 1
                if s[i - 1] == " " or i == 0:
                    break
        return count
            


        
        